src/arizona_template_compiler.erl

```diff
@@ -330,7 +336,7 @@ push_js_scripts([]) ->
 js_scripts() ->
     Scripts = case arizona_cfg:endpoint() of
         #{live_reload := true} ->
-            [live_reload_script() | required_js_script()];
+            required_js_script() ++ [live_reload_script()];
```

Comment:
Is the order relevant, here? Or why the change?

Reply:
Yes, it's relevant because `arizona-live-reload.js` depends on `arizona.js` (which is included in `required_js_script()`).

Reply:
Right. Did we think about using `defer` or `async` in that case, to minimize dependency-related issues?

Reply:
Maybe the `morphdom` lib could be loaded `async`, but I'm unsure because If the consumer needs to update the DOM immediately, this can cause an issue. I'm against using `defer` to load the `arizona.js` since we want the WebSocket connected as fast as possible. I'm for checking this type of feature when benchmarks and tests are implemented in JS files.